From 8e7cde8af909f191ba9433e6d2e931534b9ef673 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 26 Aug 2023 00:02:19 +0200
Subject: [PATCH 145/N] tests: skip a broken test

See https://github.com/python/cpython/issues/107219
Once that is fixed this commit can be removed.
This is a commit and not just an addition to the skip list, since
we still run the skipped tests in CI and in this case everything would hang.
---
 Lib/test/test_concurrent_futures.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Lib/test/test_concurrent_futures.py b/Lib/test/test_concurrent_futures.py
index 4558560..11d5001 100644
--- a/Lib/test/test_concurrent_futures.py
+++ b/Lib/test/test_concurrent_futures.py
@@ -1372,6 +1372,8 @@ class ExecutorDeadlockTest:
         executor_manager.join()
 
     def test_crash_big_data(self):
+        # https://github.com/python/cpython/issues/107219
+        return
         # Test that there is a clean exception instad of a deadlock when a
         # child process crashes while some data is being written into the
         # queue.
